src/content/migrate/5.md

```diff
@@ -103,18 +105,29 @@ Yarn: `yarn add webpack@next -D`
 - If you are using Yarn's PnP and the `pnp-webpack-plugin`, we have good news: it is supported by default now. You have to remove it from the configuration.
 - If you are using `IgnorePlugin` with a regular expression as argument, it takes an `options` object now: `new IgnorePlugin({ resourceRegExp: /regExp/ })`.
 
-If you were using WASM via import you should follow this two step process:
+If you were using WebAssembly via import you should follow this two step process:
 
 - Enable the deprecated spec by setting `experiments.syncWebAssembly: true`, to get the same behavior as in webpack 4.
 - After successful migration to webpack 5, change the `experiments` value to `experiments: { asyncWebAssembly: true }` to use the up-to-date spec for WASM integration.
 
 Reconsider `optimization.splitChunks`:
 
 - It's recommended to use either the defaults or `optimization.splitChunks: { chunks: 'all' }`.
-- When using HTTP/2 and long term caching, set `optimization.splitChunks: { chunks: 'all', maxInitialRequests: 30, maxAsyncRequests: 30, maxSize: 100000 }`.
 - When using a custom configuration, replace `name` with `idHint`.
 - It was possible to turn off the defaults by setting `optimization.splitChunks: { default: false, vendors: false }`. We don't recommend doing this, but if you really want to get the same effect in webpack 5: `optimization.splitChunks: { default: false, defaultVendors: false }`.
 
+Consider removing defaults:
+
+- Using `entry: "./src/index.js`: you can omit it, that's the default.
```

Comment:
If I remember correctly `entry: "./index.js"` is default as well. I think the priority order is `./index.js` then `./src/index.js`.

Reply:
`./index.js` is not used by default.

The default is `entry: "./src"` which would match depending on `resolve.extensions` and `resolve.mainFiles`. By default they are `resolve.extensions: [".js", ".json", ".wasm"]` and `resolve.mainFiles: ["index"]`, but e. g. typescript users often add `".ts"` so `./src/index.ts` would also be possible in this case.

Reply:
Thanks for clarifying @sokra

Is the default assignment done in the core itself? Presently webpack cli is also adding it's own default entry but if core does that already we can get rid of it from the CLI.

Reply:
```suggestion

- Using `entry: './src/index.js'`: you can omit it, that's the default.
```